registry/configuring-registry-operator.adoc

```diff
@@ -45,5 +47,5 @@ include::modules/registry-operator-config-resources-secret.adoc[leveloffset=+1]
 
 * xref:../registry/configuring-registry-storage/configuring-registry-storage-aws-user-infrastructure.adoc#configuring-registry-storage-aws-user-infrastructure[Configuring registry storage for AWS user-provisioned infrastructure]
 * xref:../registry/configuring-registry-storage/configuring-registry-storage-gcp-user-infrastructure.adoc#configuring-registry-storage-gcp-user-infrastructure[Configuring registry storage for GCP user-provisioned infrastructure]
-* xref:../registry/configuring-registry-storage/configuring-registry-storage-baremetal.adoc#configuring-registry-storage-baremetal[Configuring registry storage for bare metal]
-* xref:../registry/configuring-registry-storage/configuring-registry-storage-vsphere.adoc#configuring-registry-storage-vsphere[Configuring registry storage for vSphere]
+* xref:../registry/configuring-registry-storage/configuring-registry-storage-baremetal.adoc#configuring-registry-storage-baremetal[Configuring the registry for bare metal]
```

Comment:
Same comment as above, do you want to update titles for AWS and GCP, too?

Reply:
Updated.